models/source.yml

```diff
@@ -381,7 +381,7 @@ models:
           - not_null  
           - dbt_expectations.expect_column_values_to_be_of_type:
               column_type: numeric  
-      - name: reporting_month_number
+      - name: "माह"
         description: "Added the numbers to months for ordering in charts"
```

Comment:
is this because the columns at the source have been translated to hindi ?

Reply:
Yes